lang/cs/Org.Apache.REEF.Client/YARN/RESTClient/HttpClient.cs

```diff
@@ -33,7 +33,7 @@ internal class HttpClient : IHttpClient
         private HttpClient(IYarnRestClientCredential yarnRestClientCredential)
         {
             _httpClient = new System.Net.Http.HttpClient(
-                new HttpClientRetryHandler(new WebRequestHandler { Credentials = yarnRestClientCredential.Credentials }),
+                new HttpClientRetryHandler(new HttpClientHandler { Credentials = yarnRestClientCredential.Credentials }),
```

Comment:
HttpClientHandler is .NET Core compliant; WebRequestHandler is not. We don't actually use any functionality in WebRequestHandler that's not inherited from HttpClientHandler.